imgui_memory_editor/imgui_memory_editor.h

```diff
@@ -721,8 +721,9 @@ struct MemoryEditor
             if (data_format == DataFormat_Hex) { ImSnprintf(out_buf, out_buf_size, "%a", float64); return; }
             break;
         }
+        default:
+            IM_ASSERT(0); // Shouldn't reach
         } // Switch
-        IM_ASSERT(0); // Shouldn't reach
```

Comment:
This is the wrong thing to do here. Removing this assert will stop us from catching new case statements that won't have a return statement.